docs/FingerprintApi.md

```diff
@@ -134,6 +135,51 @@ Name | Type | Description  | Notes
 
 [[Back to top]](#) [[Back to API list]](../README.md#documentation-for-api-endpoints) [[Back to Model list]](../README.md#documentation-for-models) [[Back to README]](../README.md)
 
+# **SearchEvents**
+> SearchEventsResponse SearchEvents(ctx, limit, optional)
+Get events via search
+
+Search for identification events, including Smart Signals, using multiple filtering criteria. If you don't provide `start` or `end` parameters, the default search range is the last 7 days.  Please note that events include mobile signals (e.g. `rootApps`) even if the request originated from a non-mobile platform. We recommend you **ignore** mobile signals for such requests. 
+
+### Required Parameters
+
+Name | Type | Description  | Notes
+------------- | ------------- | ------------- | -------------
+ **ctx** | **context.Context** | context for authentication, logging, cancellation, deadlines, tracing, etc.
+  **limit** | **int32**| Limit the number of events returned.  | 
+ **optional** | ***FingerprintApiSearchEventsOpts** | optional parameters | nil if no parameters
+
+### Optional Parameters
+Optional parameters are passed through a pointer to a FingerprintApiSearchEventsOpts struct
+
+Name | Type | Description  | Notes
+------------- | ------------- | ------------- | -------------
+**optional** | ***FingerprintApiSearchEventsOpts** | optional parameters | nil if no parameters
+**optional** | ***FingerprintApiSearchEventsOpts** | optional parameters | nil if no parameters
```

Comment:
Why is it duplicated?
Why do we even have `FingerprintApiSearchEventsOpts` inside itself?